client/go/outline/electron/main.go

```diff
@@ -51,6 +50,12 @@ const (
 
 var logger = slog.New(slog.NewTextHandler(os.Stdout, nil))
 
+// The result JSON containing two error strings when "--checkConnectivity".
+type CheckConnectivityResult struct {
+	TCPErrorJson string `json:"tcpErrorJson,omitempty"`
```

Comment:
Let's call them simply `tcp` and `udp`.

Also, I think it's clearer if we always output the results, with `null` if no errors. That way we make it explicit that the test actually happened and succeeded.

Reply:
Updated, now Go will output empty string `""` if there are no errors (see the example in PR description).